script/overlayfs_chaindata.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# A tool to create/clean up the overlayfs over the chaindata directory.
+# overlayfs can be used to quickly rollback the chaindata directory
+# after testing.
+
+unset DATADIR
+unset CLEANUP
+
+PROGNAME=$(basename "$0")
+USAGE="$PROGNAME -d datadir [-c]
+
+where:
+    -d datadir: the chaindata's parent directory
+    -c:         clean up mode
+"
+
```

Comment:
```
error_exit() {
  echo "$PROGNAME: $1" >&2
  exit 1
}
```
--
```
    c) CLEANUP=true ;;
    :)
      error_exit "option requires an argument -- '$OPTARG'"
      ;;
    *)
      error_exit "bad option -$OPTARG. Try '$PROGNAME -h' for more information."
      ;;
  esac

Reply:
IMO, I don't think it's necessary. I prefer the explicitness here.